Helper/Data.php

```diff
@@ -35,8 +35,8 @@ class Data extends AbstractHelper
     const LIVE = 'live';
     const CHECKOUT_CONTEXT_URL_LIVE = 'https://checkoutshopper-live.adyen.com/checkoutshopper/';
     const CHECKOUT_CONTEXT_URL_TEST = 'https://checkoutshopper-test.adyen.com/checkoutshopper/';
-    const CHECKOUT_COMPONENT_JS_LIVE = 'https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/2.0.0/adyen.js';
-    const CHECKOUT_COMPONENT_JS_TEST = 'https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/2.0.0/adyen.js';
+    const CHECKOUT_COMPONENT_JS_LIVE = 'https://checkoutshopper-beta.adyen.com/checkoutshopper/sdk/2.1.0/adyen.js';
+    const CHECKOUT_COMPONENT_JS_TEST = 'https://checkoutshopper-beta.adyen.com/checkoutshopper/sdk/2.1.0/adyen.js';
 
```

Comment:
can we not use https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/2.1.0/adyen.js and https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/2.1.0/adyen.js?

Reply:
I checked with the checkout and they have just gone live with it yesterday so yes we can use it now. :)

Reply:
great :)